agent/xds/server.go

```diff
@@ -195,6 +199,9 @@ func (s *Server) authenticate(ctx context.Context) (acl.Authorizer, error) {
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "error fetching options from context: %v", err)
 	}
+	if options.Token == "" {
+		metrics.IncrCounter([]string{"xds", "server", "unauthenticated"}, 1)
```

Comment:
Would it make more sense to handle this in the `activeStreamCounter` as a gauge since an individual stream may call `authenticate` or `authorize` multiple times over its lifetime and would likely be double/triple/etc counted?

Reply:
Yeah that makes sense to me. Will figure out how to carry over the logic